cmd/reposervercontroller/main.go

```diff
@@ -79,7 +81,7 @@ func main() {
 	config := ctrl.GetConfigOrDie()
 	mgr, err := ctrl.NewManager(config, ctrl.Options{
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
+		Metrics:                server.Options{BindAddress: metricsAddr},
```

Comment:
LGTM, can you check if the metrices are actually being exposed after this change.